targets/nuttx-stm32f4/nuttx/config_mpu

```diff
@@ -0,0 +1,1673 @@
+#
+# Automatically generated file; DO NOT EDIT.
```

Comment:
@tadziopazur I agree with you. The preceding dot makes the file invisible in unix/osx. Could you remove dot in config file? You can change it in this PR or make another PR. In this change, please don't forget update the [guide](https://github.com/Samsung/iotjs/tree/master/targets/nuttx-stm32f4#3-configure-nuttx).